index.html

```diff
@@ -53,72 +55,130 @@
         </p>
     </div>
 
     <script type="text/javascript">
         "use strict";
 
+        var noGovDate = new Date(2022, 1, 3, 16,30); // Curse your Zero indexed dates....
+
         Date.daysSince = function (compareDate) {
             var millisecondsInDay = 24 * 60 * 60 * 1000;
             var timeDiff = new Date().getTime() - compareDate.getTime();
             return Math.floor(timeDiff / millisecondsInDay);
         }
 
-        Date.monthsSince = function(compareDate) {
+        Date.monthsSince = function (compareDate) {
             var millisecondsInDay = 24 * 60 * 60 * 1000;
             var millisecondsInMonth = millisecondsInDay * 31;
             var todaysDate = new Date();
-            if(todaysDate.getMonth() == 1){
+            if (todaysDate.getMonth() == 1) {
                 millisecondsInMonth = millisecondsInDay * 28;
-            }else if(todaysDate.getMonth() == 10 ||  todaysDate.getMonth() == 3 || todaysDate.getMonth() == 5 || todaysDate.getMonth() == 8){
+            } else if (todaysDate.getMonth() == 10 || todaysDate.getMonth() == 3 || todaysDate.getMonth() == 5 || todaysDate.getMonth() == 8) {
                 millisecondsInMonth = millisecondsInDay * 30;
             }
 
             var timeDiff = todaysDate.getTime() - compareDate.getTime();
             return Math.floor(timeDiff / millisecondsInMonth);
         }
 
-        Date.secondsSince = function(compareDate) {
+        Date.secondsSince = function (compareDate) {
             var timeDiff = new Date().getTime() - compareDate.getTime();
             return Math.floor(timeDiff / 1000);
         }
 
-        var gov = (function () {
-            var govDate = new Date(2020,0,11);
-            var daysSinceGov = Date.daysSince(govDate);
+        var noGov = (function () {
 
+            var currentRecordDays = 589;
+            var daysSinceGov = Date.daysSince(noGovDate);
 
             return {
                 totalDays: function () {
                     return daysSinceGov;
                 },
+                daysToRecord: function () {
+                    return Math.abs(currentRecordDays - daysSinceGov);
+                },
                 description: function () {
-                  return "Zero! Northern Ireland has had a government for  " + daysSinceNoGov + " days";
+                    return "It has been " + daysSinceGov + " days since Northern Ireland had a government. There have been " + currentRecordDays - daysSinceGov + " days since passing Belgium";
+                }
+            };
+        })();
+
+        var salaryCount = (function () {
+            var dailyBaseSalaryFrom2020 = 51000 / 365; // http://www.niassembly.gov.uk/your-mlas/members-salaries-and-expenses/salaries-and-expenditure-rates-2016-2021/
+
+            var salaryPerSecond = (((dailyBaseSalaryFrom2020 / 24) / 60) / 60);
+            var noOfMLAs = 90;
```

Comment:
```suggestion
            var noOfMLAs = 0;
```

Reply:
AFIK the MLA count isn't reducing despite the resignation, so the number is still needed for the salary calculations for costs incurred while NI is without a government.

Reply:
Is the salary actually going to be paid out to anyone?